package/android/src/main/cpp/AndroidLogger.cpp

```diff
@@ -8,9 +8,9 @@
 #include "MmkvLogger.h"
 #include <android/log.h>
 
-void MmkvLogger::log(const std::string& tag, const std::string& message) {
+void MmkvLogger::log(const std::string& tag, const std::string& formatString, Args... args) {
```

Comment:
same here, not a template.

we can't even template this because our implementation is in two differnt spots (ios and android). so we might not be able to support `Args...`.